chore(flags): more docs on evaluation environments (#13298)
PostHog/posthog.com: contents/docs/feature-flags/evaluation-environments.mdx

sidebar: Docs
showTitle: true
availability:
  free: false
  selfServe: boost

>   enterprise: full

Reviewer: I don't think this is used anymore and setting `selfServe: boost` is breaking the build, so maybe just delete?

---

import { ProductScreenshot } from 'components/ProductScreenshot'
import { CalloutBox } from 'components/Docs/CalloutBox'

<CalloutBox icon="IconWarning" title="Heads up!" type="caution">

> Evaluation environments are currently in alpha release and require (at least) the boost add-on (they need the resource tagging feature to function correctly). This feature may not be available in your PostHog instance yet. Contact support if you'd like early access to this feature.

Reviewer: ```suggestion
Evaluation environments are currently in alpha release and require (at least) the **Boost add-on** for the resource tagging feature to function correctly. This feature may not be available in your PostHog instance yet. Contact support if you'd like early access to this feature.
```

## Why use evaluation environments?

> ### 1. Application isolation

Reviewer: ```suggestion
### 1. Application isolation

```
